airbyte-integrations/connectors/source-file/README.md

```diff
@@ -0,0 +1,36 @@
+# Testing Source File
+
+This integration 
+
+## Necessary Credentials for tests
+
+In order to run integrations tests in this connector, you need to:
+1. Testing Google Cloud Service Storage
+    1. Download and store your Google [Service Account](https://console.cloud.google.com/iam-admin/serviceaccounts) JSON file in `secrets/gcs.json`, it should look something like this:   
```

Comment:
maybe i missed it, but how does gcs.json get imported into config.json?

Reply:
It's used in: `airbyte-integrations/connectors/source-file/integration_tests/integration_source_test.py` which are the "custom" integration tests, not the standard_tests.

The content of the JSON is copied into the configuration as a string.
In the UI, you  would need to do the same and copy/paste the content of the JSON.

Then in the source.py of this connector, we either are able to manipulate the DICT object directly once we parse that string or have to produce a temporary file with the ocntent of the json (depending on the google API we are using)